.github/workflows/code-coverage.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5.1.0
       with:
-        go-version-file: 'go.mod'
+        go-version: '1.22'
```

Comment:
Let's not do this.

Reply:
Can't make it happen without this without actions/setup-go enhancement

Reply:
Ok I figured it out. Please let me know.